ethereum/smart-contract/contracts/GatewayToken.sol

```diff
@@ -177,7 +169,6 @@ contract GatewayToken is
     function mint(
         address to,
         uint network,
-        uint expiration,
```

Comment:
I would retain the possibility for the gatekeeper to set the expiration on a token.

- Network Duration always wins.
- If no Network Duration -> set Gatekeeper passed expiration
- If no Network duration and not gatekeeper expiration -> does not expire.

Reply:
Updated to follow the above logic:

The expire time for a given gateway token is -

1) If `passExpireTimeInSeconds` is set on the network then `expiration  = block.timestamp + passExpireTimeInSeconds`

2) If `passExpireTimeInSeconds` IS NOT set on the network then `expiration` is used

3) If `passExpireTimeInSeconds` and `expiration` are 0 the pas does not expire